velox/type/Variant.cpp

```diff
@@ -248,9 +260,11 @@ std::string variant::toJson(const TypePtr& type) const {
       folly::json::escapeString(str, target, getOpts());
       return target;
     }
-    case TypeKind::HUGEINT:
-      VELOX_CHECK(type && type->isLongDecimal());
-      return DecimalUtil::toString(value<TypeKind::HUGEINT>(), type);
+    case TypeKind::HUGEINT: {
+      VELOX_CHECK(type && type->isLongDecimal()) {
```

Comment:
Let's move the check for null type to the start of this function. Should this be user check like others?

Also, let's remove check for null type from case TypeKind::BIGINT branch.

Reply:
Removed the check for null type for BIGINT and HUGEINT cases.